mitodl/bootcamp-ecommerce
# Initial work on hubspot contact sync

#### What are the relevant tickets?
closes #297 

#### What's this PR do?
This PR adds hubspot integration and contact sync. Contacts are either synced through a management command or automatically when the backend gets a webhook.

#### How should this be manually tested?
Make your local copy of the app publicly available and point test webhooks at it from Smapply. Make those webhooks fire and check hubspot for the contact.

Alternatively manually add the `smapply_demographic_data` dictionary from hubspot/conftest.py to a Profile's smapply_demographic_data. Then manually run `sync_contact_with_hubspot(profile_id)` and check hubspot for your contact.
